### DISCUSSION TITLE
Documentation for the OpenVINO AI plugins for Audacity

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [intel/openvino-plugins-ai-audacity#127] [Show and tell]
As someone who has written a fair chunk of the Audacity Manual (and I remain as the maintainer, for now at least) I am **_delighted_** to see that there is useful documentation present for this set of AI plugins:

https://github.com/intel/openvino-plugins-ai-audacity

**Comment by RyanMetcalfeInt8** [comment]
@petersampsonaudacity -- awesome, thanks! Feel free to point out any gaps / things that aren't clear.

Regards,
Ryan

**Comment by insaneninja117** [comment]
I've used this plugin on two different CPUs:

1. An i5-560m (Westmere, 1.5th Gen Core i)
2. An i7-6700k (Skylake, 6th Gen Core i)

The 6700k runs the plugin with no issues and at high speed, but the 560m returns an error code and the preferences menu shows that the OpenVINO module failed to load. Running Audacity through Intel SDE with the emulated architecture set to Sandy Bridge (2nd Gen Core i) everything loads fine, although music separation runs at a glacial pace presumably because of the need to emulate the AVX1 instructions that were introduced with Sandy Bridge.

I would recommend that the documentation state that the user requires a CPU that supports AVX1.

Also, this was done on a portable version of Audacity 3.4.2, and running on Windows 7 on both machines, so even though Win 7 was dropped ages ago, both Audacity 3.4.2 and the OpenVINO module works without issue on Win 7.

**Comment by RyanMetcalfeInt8** [comment]
Hi @insaneninja117,

Thanks for the feedback. Yes, in general I am missing information about system requirements. As this plugin depends on OpenVINO, we essentially have the same system requirements as that framework -- which I believe is "6th Gen core i" or later. 

Anyway, thanks, I'll add this information to release notes and readme.

Regards,
Ryan

**Reply by MarkR-A** [reply]
Hi Ryan, first of all thanks for all your amazing work! Do you know when, if at all, it will be compatible with Intel Xeon CPUs?
